tailcfg/tailcfg.go

```diff
@@ -317,6 +318,9 @@ type Node struct {
 	//    "https://tailscale.com/cap/file-sharing"
 	Capabilities []NodeCapability `json:",omitempty"`
 
+	// CapMap is a map of capabilities to their raw values.
```

Comment:
say more about the values and how they can be nil? and the relation between this and the Capabilities slice?